server/vulnerabilities/msrc/io/github.go

```diff
@@ -31,15 +31,17 @@ type GitHubAPI interface {
 }
 
 type GitHubClient struct {
+	ctx        context.Context
```

Comment:
Why have a context on the struct as opposed to passing context as the first argument to `Bulletins`?  This is against the general advice from the context package. From https://pkg.go.dev/context

> Do not store Contexts inside a struct type

Reply:
N00b mistake, thanks!

Reply:
Done